src/components/Disputes/PrecedenceCampaign/PrecedenceCampaignModal.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { Button, GU, Modal, textStyle, useTheme, useViewport } from '@aragon/ui'
+
+import pcModalSvg from './assets/PCModal.svg'
+
+function PrecedenceCampaignModal({ opened, onClose }) {
+  const { width, height } = useViewport()
+  const theme = useTheme()
+
+  const verticalMode = width < 1000
+  const modalHeight = verticalMode
+    ? height - 40
+    : Math.max(700, Math.min(620, height - 40))
+
+  return (
+    <div>
+      <Modal
+        closeButton={false}
+        padding={0}
+        visible={opened}
+        width={Math.min(1055, width - 40)}
+        css={`
+          z-index: 4;
+        `}
+      >
+        <div
+          css={`
+            display: grid;
+            grid-template-${verticalMode ? 'rows' : 'columns'}: 1fr 1fr;
+            height: ${modalHeight}px;
+          `}
+        >
+          {verticalMode ? (
+            <div
+              css={`
+                position: relative;
+              `}
+            >
+              <div
+                css={`
+                  background: url(${pcModalSvg}) 50% 50% / cover no-repeat;
+                  position: absolute;
+                  top: 0;
+                  bottom: 0;
+                  left: 0;
+                  right: 0;
+                `}
+              />
+            </div>
+          ) : (
+            <div
+              css={`
+                display: flex;
+                align-items: flex-end;
+                height: 100%;
+              `}
+            >
+              <img src={pcModalSvg} height={modalHeight * 0.85} />
+            </div>
+          )}
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+              height: 100%;
+            `}
+          >
+            <div
+              css={`
+                text-align: ${verticalMode ? 'center' : 'left'};
+                padding: 0px ${7 * GU}px;
+              `}
+            >
+              <p
+                css={`
+                  ${textStyle('body2')};
+                  color: ${theme.contentSecondary};
+                  text-transform: uppercase;
+                `}
+              >
+                Aragon court
```

Comment:
Let's capitalize the name: **Court** 😄

Reply:
Sure, in any case note that the whole text is transformed to uppercase :D